features/telemetry/adapters/registry/pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <parent>
+    <groupId>org.opennms.features.telemetry</groupId>
+    <artifactId>org.opennms.features.telemetry.adapters</artifactId>
+    <version>22.0.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.opennms.features.telemetry.adapters</groupId>
+  <artifactId>org.opennms.features.telemetry.adapters.registry</artifactId>
+  <name>OpenNMS :: Features :: Telemetry :: Adapters :: Registry</name>
+  <packaging>bundle</packaging>
+  <build>
```

Comment:
The `maven-bundle-plugin` setting is duplicated a lot. We may want to define it in a parent module instead.

Reply:
parent module is adapters which just includes other modules. There may be other adapters which may not be osgi bundles. I will keep  it that way.  Also good to have bundle info in it's own pom.xml I feel.

Reply:
The definition here, only extends maven, which allows defining bundles. It does not make all modules bundles. The type is defined by the `<type>bundle</type>` statement. Therefore I am still for moving it to a parent module.